packages/site/src/components/Navigation/TopNav.tsx

```diff
@@ -182,19 +182,35 @@ function ActionMenu({ actions }: { actions?: SiteManifest['actions'] }) {
   );
 }
 
-function HomeLink({ logo, logoText, name }: { logo?: string; logoText?: string; name?: string }) {
+function HomeLink({
+  logo,
+  logoDark,
+  logoText,
+  name,
+}: {
+  logo?: string;
+  logoDark?: string;
+  logoText?: string;
+  name?: string;
+}) {
   const Link = useLinkProvider();
   const baseurl = useBaseurl();
   const nothingSet = !logo && !logoText;
   return (
     <Link
-      className="flex items-center dark:text-white w-fit ml-3 md:ml-5 xl:ml-7"
+      className="flex items-center ml-3 dark:text-white w-fit md:ml-5 xl:ml-7"
       to={withBaseurl('/', baseurl)}
       prefetch="intent"
     >
       {logo && (
-        <div className="dark:bg-white dark:rounded p-1 mr-3">
```

Comment:
I think we might want another branch/if-statement here for the existing behaviour when the dark logo is not defined. I think that should still have the white background?

Reply:
Yeah, I was thinking about that too.

Probably something like `logoDark ?? <div className="..."></div>` will work.

Reply:
The `dark:hidden` class name also needs to be added depending as well.

We have been using something like:

```jsx
<img className={classNames('shared-classes', { 'darkhidden': !!logoDark })} />
```

Reply:
Oh, that looks much cleaner. Thanks Rowan!

Reply:
What do you think of https://github.com/executablebooks/myst-theme/pull/130/commits/2e3b14fb79e55448fc9b50b2c518e8047cee912c